arrows/core/associate_detections_to_tracks_threshold.h

```diff
@@ -31,48 +31,44 @@ class KWIVER_ALGO_CORE_EXPORT associate_detections_to_tracks_threshold
   virtual ~associate_detections_to_tracks_threshold() noexcept;
 
   /// Get this algorithm's \link vital::config_block configuration block \endlink
-  /**
-   * \returns \c config_block containing the configuration for this algorithm
-   *          and any nested components.
-   */
+  ///
+  /// \returns \c config_block containing the configuration for this algorithm
+  ///          and any nested components.
```

Comment:
Should there be extra white space on this line?

Reply:
This is a good point which we should discuss.  This happens in **many** places throughout the code.  If we are to address this, I think it should be applied across all of Kwiver and would make a good PR in and of itself.

Reply:
I'm not sure what "extra white space" is being claimed? If you mean that `and` is indented to after `\return`, that is both expected and [ought to be] prescribed, *exactly* like aligning continued function arguments after the function name. (Conceptually, `\return` here is a function and the text is the arguments thereto.)